Casks/w/webex.rb

```diff
@@ -10,11 +10,12 @@
   homepage "https://www.webex.com/"
 
   livecheck do
-    url :url
-    strategy :extract_plist
+    url "https://help.webex.com/en-us/mqkve8"
```

Comment:
```suggestion
    url "https://help.webex.com/en-us/article/mqkve8"
```

The server redirects to `/en-us/article/mqkve8`, so we should use that. The URL in the browser ends up being longer but I think that happens client-side, as the aforementioned URL returns a 200 and the response body contains the expected HTML.